lib/memcachir.ex

```diff
@@ -131,9 +136,19 @@ defmodule Memcachir do
   end
   defp execute(fun, node, args) do
     try do
-      :poolboy.transaction(node, &apply(fun, [&1 | args]))
+      node
```

Comment:
These changes are all about exposing the node instead of the pool to this function. We need the actual node for the unavailable node tracking rather than the pool like we were receiving before.